CIPs/0003.md

```diff
@@ -0,0 +1,134 @@
+# CIP [0003]: Identity Claims and Metadata
+
+- Date: 2019-09-19
+- Author: @timmoreton
+- Status: WIP
+
+## Overview
+
+This CIP describes a general mechanism that allows principals on the Celo platform to claim real-world identities and maintain rich, unverified metadata. Metadata may be public, or optionally, encrypted and accessible only to the Celo Foundation. Third parties can verify these identity claims, and use the associated unverified metadata as appropriate. 
```

Comment:
I would caution about putting responsibilities on the Celo foundation:
1/ not yet clear what the roles and responsibilities of the foundation will be, recommendations received from other projects is to limit the scope to resource allocation only (no process or day-to-day management, which should be outsourced)
2/ I'd be in favor of planning for a day where the foundation doesn't even need to exist anymore (a la bitcoin)